### DISCUSSION TITLE
2FA I'm not receiving SMS

### DISCUSSION THREAD

**Discussion by fortytwodev** [community/community#72221] [New to GitHub]
Every day I am asked about 2FA verification of my account, but I do not receive an SMS code from github. When I click on the resend button, an error appears that cannot send the code to my number (country code in the white list)
Because of this, I skip step 2FA verification every day (doing it everyday is annoying).

**Comment by Xrenya** [comment]
You can use 2FA Authenticator (2FAS) app instead of using SMS.
